src/helpers/releasesInfo.js

```diff
@@ -4,6 +4,17 @@ import constants from "./constants";
 export const retrieveReleases = gitHubURL =>
     new Promise((resolve, reject) => {
         const repositoryName = gitHubURL.split("github.com")[1];
+        const lsReleasesData = localStorage.getItem(constants.LOCAL_STORAGE.RELEASES_DATA) || "";
+        const releasesData = lsReleasesData.split(";")[0];
```

Comment:
If the entry is not there then wont this be null? Does this not cause error say in a fresh page? 
Did you try this after clearing the `localStorage`?

Reply:
Yes tried it after deleting the local storage manually so that it gets written again.  The `||""`  on line 7 will set the `lsReleasesData` to an empty string so there will not be any errors